demonstrations/tutorial_vqe_uccsd_obs.py

```diff
@@ -0,0 +1,253 @@
+r"""
+VQE simulations in different sectors of the spin quantum number
+===============================================================
+
+.. meta::
+    :property="og:description": Find the lowest-energy states of a Hamiltonian in different
+        sector of the spin quantum number using the variational quantum eigensolver
+        algorithm in PennyLane.
+    :property="og:image": https://pennylane.ai/qml/_images/pes_h2.png
+
+Quantum computers offer a promising avenue to perform first-principles simulations of the
+electronic structure of molecules and materials that are currently intractable using classical
+high-performance computers. In particular, the Variational Quantum Eigensolver (VQE) algorithm
+:ref:`[1, 2]<vqe_references>` has proven to be a valuable quantum-classical computational
+approach to find the lowest-energy eigenstate of the electronic Hamiltonian by using Noisy
+Intermediate-Scale Quantum (NISQ) devices. (WILL ADD SOME REFERENCES HERE)
+
+In the absence of `spin-orbit coupling <https://en.wikipedia.org/wiki/Spin-orbit_interaction>`_ the
+electronic Hamiltonian matrix is block diagonal in the total spin quantum numbers. In other words,
+one can expand the many-electron wave function of the molecule as a linear
```

Comment:
```suggestion
we can expand the many-electron wave function of the molecule as a linear
```
The passive voice is too formal for a tutorial

Reply:
I find this sentence very loaded and technical, which makes it difficult to follow. I think the paragraph reads better if we remove it

Reply:
Agree. I rephrase it like this:

"This allows to compute the energy spectrum of the Hamiltonian in a given sector of
:math:`S_z`."